zeebe/exporters/elasticsearch-exporter/src/main/java/io/camunda/zeebe/exporter/ElasticsearchMetrics.java

```diff
@@ -53,6 +54,14 @@ public ElasticsearchMetrics(final int partitionId, final MeterRegistry registry)
             .description("Number of failed flush operations")
             .tags(PARTITION_LABEL, partitionIdLabel)
             .register(meterRegistry);
+
+    recordBufferLifetime =
+        Timer.builder(meterName("record.buffer.lifetime"))
```

Comment:
🙃 Naming is hard. Of course the name is discussable and happy for any input.

Reply:
Thinking about this we might want to call it flush latency 🤔 thinking about the discussion here https://github.com/camunda/camunda/pull/21297#discussion_r1725204578

Reply:
Flush latency sounds good to me. Alternate: record buffering latency.